README.md

```diff
@@ -14,16 +14,23 @@ Building for Fedora
 
 Packages needed for the build, or the build requires:
 
+* make
 * check-devel
 * cmake
 * expat-devel
 * gcc
 * libsolv-devel
 * libsolv-tools
+* librepo-devel
 * python-devel (or python3-devel for Python 3 build)
 * python-nose (or python3-nose for Python 3 build)
 * rpm-devel
 * zlib-devel
+* glib2-devel
+* gtk-doc
+* gobject-introspection-devel
+* redhat-rpm-config
```

Comment:
@Conan-Kudo @jsilhan Sorry, should have clarified this. The `redhat-rpm-config` package is required (starting from Fedora 23 at least) due to the new [hardening change](https://fedoraproject.org/wiki/Changes/Harden_All_Packages). I don't think this is a req (yet) for RHEL/Centos, so I can leave it off if you'd like.